### PR TITLE
Issue #3 fix

### PR DESCRIPTION
**Pull request by lfreina**
Namespace fix for issue #3.

Selector was also added.

### CODE REVIEW

File: scripts/kube/build/airflow.yaml [hunk 1]
```diff
@@ -30,7 +36,7 @@ metadata:
   namespace: airflow-example
   name: airflow
 rules:
-  - apiGroups: [""] # "" indicates the core API group
+  - apiGroups: [""]   # "" indicates the core API group
```
Comment (bhavaniravi) [inline scripts/kube/build/airflow.yaml:39]: Remove spaces

File: scripts/kube/build/airflow.yaml [hunk 2]
```diff
@@ -44,11 +50,12 @@ metadata:
   namespace: airflow-example
 subjects:
   - kind: ServiceAccount
-    name: airflow # Name of the ServiceAccount
+    name: airflow   # Name of the ServiceAccount
```
Comment (bhavaniravi) [inline scripts/kube/build/airflow.yaml:53]: Remove spaces

File: scripts/kube/build/airflow.yaml [hunk 3]
```diff
@@ -127,7 +137,7 @@ spec:
             - name: airflow-logs
               mountPath: /root/airflow/logs
         - name: scheduler
-          image:
+          image:
```
Comment (bhavaniravi) [inline scripts/kube/build/airflow.yaml:140]: Same as above

**Review by bhavaniravi — CHANGES_REQUESTED** [review]

File: scripts/kube/build/airflow.yaml (hunk 1 above)
Comment (lfreina) [reply scripts/kube/build/airflow.yaml]: YAML lint was complaining about those spaces but I can remove them. No worries.

**Comment by lfreina** [comment]
I also took a split into line 57 as the comment was too long. ;D This should do.

**Pull request merged**